docker/Dockerfile

```diff
@@ -1,52 +1,18 @@
-################################################################################
-# Build stage 0 `builder`:
-# Install OpenSearch Benchmark from source inside a virtualenv
-################################################################################
-
-FROM python:3.11.2-slim as builder
-
-RUN apt-get -y update && \
-    apt-get install -y curl git gcc && \
-    apt-get -y upgrade && \
-    rm -rf /var/lib/apt/lists/*
-
-WORKDIR /opensearch-benchmark/
-COPY ./setup.py /opensearch-benchmark/
-COPY ./setup.cfg /opensearch-benchmark/
-COPY ./version.txt /opensearch-benchmark/
-COPY ./README.md /opensearch-benchmark/
-COPY ./MANIFEST.in /opensearch-benchmark/
-COPY ./osbenchmark/ /opensearch-benchmark/osbenchmark/
-COPY ./scripts/ /opensearch-benchmark/scripts/
-
-RUN python3 -m venv /opensearch-benchmark/venv
-ENV PATH="/opensearch-benchmark/venv/bin:$PATH"
-
-WORKDIR /opensearch-benchmark
-# Wipe away any lingering caches, copied over from the local machine
-RUN find ./opensearch-benchmark -name '__pycache__' -o -name '*.pyc' | xargs rm -f
-RUN python3 -m pip install --upgrade pip setuptools wheel
-RUN python3 -m pip install /opensearch-benchmark
-
-################################################################################
-# Build stage 1 (the actual OpenSearch Benchmark image):
-# Copy OpenSearch Benchmark from stage 0 and fix permissions to support randomized UIDs
-# Define VOLUME for ~/.benchmark
-################################################################################
-
 FROM python:3.11.2-slim
+ARG VERSION
+ARG BUILD_DATE
 
 ENV BENCHMARK_RUNNING_IN_DOCKER True
 
 RUN apt-get -y update && \
-    apt-get install -y curl git pbzip2 pigz && \
+    apt-get install -y curl git gcc pbzip2 pigz && \
```

Comment:
Where is `gcc` used -- why didn' t it appear in the previous Dockerfile?

Reply:
I included it since the base layer image had it installed in the `Docker-development` file. 
https://github.com/opensearch-project/opensearch-benchmark/blob/61d47687f38f2bda85842eeab2af735e63e48cee/docker/Dockerfile#L9